src/mappings/gns.ts

```diff
@@ -166,6 +167,7 @@ export function handleSubgraphPublished(event: SubgraphPublished): void {
 
   // Update subgraph
   let subgraph = createOrLoadSubgraph(subgraphID, event.params.graphAccount, event.block.timestamp)
+  let oldVersionID = subgraph.currentVersion
 
   versionID = joinID([subgraphID, subgraph.versionCount.toString()])
   subgraph.currentVersion = versionID
```

Comment:
Should `let oldVersionID = subgraph.currentVersion` line of code be after this line? (After we set `subgraph.currentVersion`) ?

Reply:
We actually want to maintain the previous version id (before we set the new one, even if it's null) that's why I do it before ;)